src/charm.py

```diff
@@ -220,8 +221,10 @@ def _command(self) -> str:
             args.append(f"--web.external-url={external_url}")
 
         # enable remote write if an instance of the relation exists
-        if self.model.relations[DEFAULT_REMOTE_WRITE_RELATION_NAME]:
-            args.append("--enable-feature=remote-write-receiver")
+        # FIXME for some reason this does not work
+        # FIXME if user changes relation name this will break anyway
+        # if self.model.relations[DEFAULT_REMOTE_WRITE_RELATION_NAME]:
+        #     args.append("--enable-feature=remote-write-receiver")
```

Comment:
User will not change the relation name on this side of the charm, so that should be fine, but instead of `DEFAULT_REMOTE_WRITE_RELATION_NAME` it should prolly set the relation name to a var on self and check for that.

Reply:
You fixed it :) resolving